### DISCUSSION TITLE
Correct configuration for SP17S005P17S40 with Li-Ion 18650 cells

### DISCUSSION THREAD

**Discussion by rmcbc** [mr-manuel/venus-os_dbus-serialbattery#90] [Q&A]
Hi,

I'm using with my Multiplus II 48/3000/35, while waiting for my new battery 16S LiFePo4, I'm using one home built 14S14P 50Ah with a JBD SP17S005P17S40 that works ok. And the BMS can be accessed by this driver using Bluetooth, at least I see all parameters from BMS correct.
My problem is the best configuration that I should do to used on the correct way:

- Max voltage: 58.1V and Low voltage: 43.50
- Max cell voltage: 4.15v and low cell voltage: 3.1v
- Max charge amperage: 20A and max discharge amperage: 25A
- SOC: 100% at 58.1V pack or 4.15v cell

Because Li-Ion cells have a different charge curve, I'm having trouble to configure this driver.
Does anyone did a configuration similar to mine and can help me?
Thanks in advance for your support.

**Comment by mr-manuel** [comment]
You find everything you need in the documentation.
